dbatools.psd1

```diff
@@ -67,6 +67,7 @@
 	
 	# Functions to export from this module
 	FunctionsToExport = @(
+		'Add-DbaToolsToProfile',

```

Comment:
While @potatoqualitee is the only individual that can alter the psd1 file I don't think this needs to be a public function. Internal is sufficient.